artifactory/commands/transferfiles/utils.go

```diff
@@ -296,14 +300,17 @@ func GetThreads() int {
 // Number of threads in the settings files is expected to change by running a separate command.
 // The new number of threads should be almost immediately (checked every waitTimeBetweenThreadsUpdateSeconds) reflected on
 // the CLI side (by updating the producer consumer if used and the local variable) and as a result reflected on the Artifactory User Plugin side.
-func periodicallyUpdateThreads(pcWrapper *producerConsumerWrapper, doneChan chan bool, buildInfoRepo bool) {
+// This method also read '~/.jfrog/transfer/stop' file and interrupt the transfer if exist.
```

Comment:
...also read... --> ...also looks for...
interrupt --> interrupts
exist --> exists.